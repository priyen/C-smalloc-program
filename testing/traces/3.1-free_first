m 0 16
m 1 32
m 2 64
m 3 128
f 0
