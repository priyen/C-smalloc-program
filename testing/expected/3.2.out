Total free mem: 261968
Total allocated mem: 176
Number of allocated blocks: 3
